static/sass/partials/_header.scss

```diff
@@ -1,19 +1,129 @@
 /* SASS file for header. */
 header {
+	/*Sitename and menu.*/
+	.sitename-wrapper {
+		.desktop-menu {
+			display: none;
+		}
+		border-bottom: 0.2px solid $border-btn-color;
+			.text-logo {
+				text-transform: uppercase;
+				text-align: left;
+				margin: 15px 0 22px 0;
+				font-family: $text-font;
+				padding-top: 4px;
+				margin-left: 5px;
+					a {
+						font-size: 28px;
+						font-weight: bold;
+						color: $primary-color;
+						text-decoration: none;
+					}
+				.text-logo-second {
+					color: $secondary-color;padding: 0;
+				}
+			}
+		span {
+			margin-top: 17px;
+			padding-left: 27px;
+		}
+		.menu-wrapper {
+			text-transform: uppercase;
+	    height: 100%;
+	    width: 0;
+	    position: fixed;
+	    z-index: 1;
+	    top: 0;
+	    right: 0;
+	    background-color:$primary-color;
+	    overflow-x: hidden;
+	    transition: 0.5s;
+	    padding-top: 0px;
+	    text-align:center;
+	    	.close-button  {
+	    		font-size: 45px;
+	    		float: right;
+	    		padding: 0px;
+    			a {
+    				text-decoration: none;
+						color: $primary-bg;
+    			}
+	    	}
+				ul.menu-list {
+					list-style-type: none;
+					display: block;
+					font-size: 25px;
+					text-align: left;
+					padding: 0px;
+	   		 	margin-bottom: 0;
+	    		width: 100%;
+	    		text-transform: capitalize;
+						a {
+					    text-decoration: none;
+							color: $primary-bg;
+							font-family: $text-font;
+							display: block;
+							padding: 7px;
+							padding-left: 40px;
+							padding-bottom: 10px;
+						}
+						a:hover {
+							color: $primary-bg;
+						}
+					li {
+						border-bottom: 1px solid rgba(255, 255, 255,0.2);
+					}
+					li:hover {
+						background-color: rgba(255, 255, 255,0.2);
+					}	
+				}
+					.register-button {
+						text-transform: capitalize;
+							a {
+								font-size: 25px;
+								font-family: $text-font;
+								color: $primary-bg;
+								text-decoration: none;
+								border: 1px solid $primary-bg;
+								background-color: $secondary-color;
+								padding: 8px;
+	    					padding-right: 60px;
+	    					padding-left: 60px;
+	    					border-color: $secondary-color;
+							}
+							a:hover {
+								color:$primary-bg;
+								background-color: $submit-btn-hover;
+							}
+					}
+			}
+	}
   .logo {
     text-align: center;
     margin-top: 40px;
     img {
       height: 350px;
     }
   }
-  .description {
+  .description {	
     margin-top: 40px;
   }
 }
 
 @media (min-width: 640px) {
```

Comment:
Please add a small comment to describe what kind of device this breakpoint is targeting.

Reply:
done